plugins/amongus/amongus.cpp

```diff
@@ -12,6 +12,10 @@
 
 #include <sstream>
 
+#if _MSC_VER == 1900
+#include <memory>
+#endif
```

Comment:
This should be included always not only for a specific compiler (version)
Same goes for the other `<memory>` includes